.cirrus.yml

```diff
@@ -8,8 +8,9 @@ env:
 
     DEBIAN_NAME: "debian-11"
 
-    UBUNTU_NAME: "ubuntu-22.04"
-    UBUNTU_PRIOR_NAME: "ubuntu-20.04"
+    UBUNTU_1804_NAME: "ubuntu-18.04"
+    UBUNTU_2004_NAME: "ubuntu-20.04"
+    UBUNTU_2204_NAME: "ubuntu-22.04"
```

Comment:
Drop versioning from the field names. We use `DISTRO_NAME` and `DISTRO_PRIOR_NAME` as it makes ongoing maintenance easier when we need to update cirrus.yml. The reason we have 8/9 in the Centos fields is because it's literally baked into the Stream names, but if we need to update images at any point we don't need to rename the fields in multiple places. We just need to update the string in `*_NAME` and the relevant image in `*_IMAGE_NAME`.

For this, we could use something like `UBUNTU_*`, `UBUNTU_PRIOR_*`, and `UBUNTU_LTS_*`.

Reply:
ack, will think about how I can refactor, although all of these versions are LTS releases, with 5 years of support from initial release